Userland/Utilities/diff.cpp

```diff
@@ -0,0 +1,40 @@
+#include <LibCore/ArgsParser.h>
+#include <LibCore/File.h>
+#include <LibDiff/Generator.h>
+
+int main(int argc, char** argv)
+{
+
+    Core::ArgsParser parser;
+    char const* filename1;
+    char const* filename2;
+
+    parser.add_positional_argument(filename1, "First file to compare", "file1", Core::ArgsParser::Required::Yes);
+    parser.add_positional_argument(filename2, "Second file to compare", "file2", Core::ArgsParser::Required::Yes);
+    parser.parse(argc, argv, Core::ArgsParser::FailureBehavior::PrintUsageAndExit);
+
```

Comment:
pledge and unveil?

Reply:
Added pledge, but I don't know if unveiling is necessary? (Following the example of `cat.cpp`)